.release-notes/4307.md

```diff
@@ -0,0 +1,15 @@
+## Enable building of libponyc-standalone on Windows
+
+We now ship a "standalone" version of libponyc for Windows. libponyc-standalone allows applications to use Pony compiler functionality as a library. The standalone version contains "all dependencies" needed in a single library. This library doesn't contain C or C++ runtime libs, as those get linked into every Pony program during compilation on Windows.
+
+An example pony program linking against it on Windows would need to look like this:
+
+```pony
+use "lib:ponyc-standalone" if posix or osx or windows
+use "lib:c++" if osx
```

Comment:
is a libc++ not needed for windows? are we statically linking one?

Reply:
Yes, those are always statically linked. I was testing this successfully on my Windows 10 machine.